pkg/operator/controllers/alertwebhook/alertwebhook_controller.go

```diff
@@ -76,7 +76,7 @@ func (r *AlertWebhookReconciler) setAlertManagerWebhook(ctx context.Context, add
 				continue
 			}
 
-			if name, ok := r["name"].(string); !ok || name != "null" {
+			if name, ok := r["name"].(string); !ok || name != "Default" {
```

Comment:
```suggestion
			if name, ok := r["name"].(string); !ok || (name != "null" && name != "Default") {
```

Reply:
Thanks for the suggestion @jim-minter . I have modified the code and pushed a new version of the PR which handles both new and old clusters and updated the unit test accordingly.